api/__tests__/app.test.js

```diff
@@ -0,0 +1,54 @@
+const supertest = require('supertest');
+const db = require("../../db/connection");
+const seed = require('../../db/seeds/seed');
+const app = require('../app');
+const data= require('../../db/data/test-data')
+
+beforeAll(() =>{
+    return seed(data)
+});
+
```

Comment:
I see this is at the bottom - I'd be inclined to add it to the top so that other developers can see that you've used it - it's an event listener so will be invoked wherever it is.